### DISCUSSION TITLE
Mouse copy looks not work

### DISCUSSION THREAD

**Discussion by x2c3z4** [adoyle-h/one.nvim#19] [Q&A] [answered]
Select texts with mouse, and Command-C to copy, nothing is copied.

**Comment by adoyle-h** [comment, marked as answer]
I think it is not a usual cases in vim. And I tried `<D-c>` and `<cmd>c`, neither works. The ⌘ key does not trigger.

## Reason

`<⌘-...>` shortcuts never send key sequences to vim and nvim. Because these shortcuts are OS-level or App-level.
For example, I use iTerm2 as a pseudo terminal. The `<⌘-c>` is captured by iTerm2.

<img width="400px" src="https://github.com/adoyle-h/one.nvim/assets/1998490/cdc7e902-eba7-4afb-9d4b-5a1eaaa65028">

Also see https://github.com/neovim/neovim/issues/14181

## Solution

Do not create keymaps with ⌘. You can use `<leader>y` to copy current line into system clipboard. And `<leader>P` to paste from system clipboard. Please see the [keymaps.md#Copy and Paste](https://github.com/adoyle-h/one.nvim/blob/a351f9253aac48eba2624163496fb0803d72714e/doc/usage/keymaps.md#copy-and-paste).

**Comment by x2c3z4** [comment]
I don't create keymaps with ⌘, and I also use iTerm2 to ssh to remote server, `<leader> y `works well, so this solution is acceptable.

Millions of thanks.
